src/api/public/apidocs-new/components/schemas/person.yaml

```diff
@@ -0,0 +1,18 @@
+type: object
+properties:
+  login:
+    type: string
+  email:
+    type: string
```

Comment:
It would be nice to have some value example here as it allows users to try it out in the web interface without having to type anything. Something like this:

https://github.com/openSUSE/open-build-service/blob/822926b893b11957161e9d7413a05a1fe85e1f1a/src/api/public/apidocs-new/components/schemas/configuration.yaml#L51-L53

Reply:
I added examples for the main attributes